benchmarking/cluster3nodes/configuration/log-config-explorer.yaml

```diff
@@ -2,6 +2,7 @@
 
 NetworkName: mainnet
 RequiresNetworkMagic: RequiresNoMagic
+GenesisHash: 5fbffc20a6ee41093a0f6a44e6110b66a69c7cc591c714075a6c537c360e32b6
```

Comment:
`GenesisHash` shouldn't be required in the `.yaml` files anymore.

Reply:
the explorer still depends on this.